core/keys/win/modifier_key.talon-list

```diff
@@ -0,0 +1,10 @@
+list: user.modifier_key
+os: windows
+os: linux
+-
+alt: alt
+control: ctrl
+shift: shift
+super: super
```

Comment:
Might be worth adding a comment to indicate that the super key is the Windows key.